src/main/kotlin/com/ampnet/reportserviceeth/service/data/SingleTransactionSummary.kt

```diff
@@ -1,9 +1,23 @@
 package com.ampnet.reportserviceeth.service.data
 
-data class SingleTransactionSummary(
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+
+class SingleTransactionSummary(
     val transaction: Transaction,
     val userInfo: UserInfo,
     val translations: Translations,
-    val logo: String = DEFAULT_LOGO,
-    val period: String? = null
-)
+    val txCreatedAt: LocalDateTime,
+    val logo: String = DEFAULT_LOGO
+) {
+    private val locale: Locale = if (userInfo.language.isBlank()) {
+        Locale.ENGLISH
+    } else {
+        Locale.forLanguageTag(userInfo.language)
+    }
+    val period: String = formatToYearMonthDay(txCreatedAt)
+
+    private fun formatToYearMonthDay(date: LocalDateTime): String =
+        date.format(DateTimeFormatter.ofPattern(DATE_FORMAT).withLocale(locale))
```

Comment:
This function is duplicated from `TransactionsSummary`. Write it as an extension of LocalDateTime.